test/e2e/framework/metrics_util.go

```diff
@@ -117,9 +117,9 @@ var InterestingApiServerMetrics = []string{
 }
 
 var InterestingControllerManagerMetrics = []string{
-	"garbage_collector_event_processing_latency_microseconds",
-	"garbage_collector_dirty_processing_latency_microseconds",
-	"garbage_collector_orphan_processing_latency_microseconds",
+	"garbage_collector_event_queue_latency",
+	"garbage_collector_dirty_queue_latency",
+	"garbage_collector_orhan_queue_latency",
```

Comment:
The name is defined by the workqueue util: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/workqueue/prometheus/prometheus.go#L56-L57.

We used to set a custom metrics for garbage collector's queues, but refactored to reuse the workqueue's metrics.